packages/time-picker/test/keyboard-navigation.test.js

```diff
@@ -12,13 +13,6 @@ describe('keyboard navigation', () => {
     inputElement = timePicker.inputElement;
   });
 
-  afterEach(async () => {
-    if (comboBox.opened) {
-      comboBox.opened = false;
```

Comment:
This isn't needed because the combo-box closes itself on detach:

https://github.com/vaadin/web-components/blob/afe2335f40764f3cde2c289d89021f5d6e331c7e/packages/combo-box/src/vaadin-combo-box-mixin.js#L355-L359